custom_weight_gh_config.yaml

```diff
@@ -0,0 +1,42 @@
+# Configuration file for standard GraphHopper routers.
+# 1 profile loaded with default speeds is defined for each vehicle type
+
+graphhopper:
+  datareader.file:
+  graph.location: transit_data/graphhopper
+  routing.ch.disabling_allowed: true
+  routing.max_visited_nodes: 1500000
+  graph.flag_encoders: car,bike,foot,truck
+
+  # Profiles specifying vehicle and weightings for each mode type.
+  profiles:
```

Comment:
This is used for running/building the sandbox. Note: no contraction hierarchy building is specified here, and `weighting: custom` + `custom_model_file: empty` is how you tell graphhopper "I'll send a custom model to you per-request, vs specifying one statically in a yaml"